package.json

```diff
@@ -13,10 +13,10 @@
   },
   "dependencies": {
     "amqp": "0.2.6",
-    "async": "0.2.7",
-    "lodash": "4.17.5",
+    "async": "1.4.0",
+    "lodash": "4.17.10",
     "node-uuid": "^1.4.4",
-    "rc": "0.1.1"
+    "rc": "1.2.7"
```

Comment:
no breaking changes in our use of rc, with our usage being supported by updated version on rc lib